tests/nightly/test_large_array.py

```diff
@@ -351,6 +351,69 @@ def test_topk():
     l = nd.topk(b, k=1, axis=-1, dtype=np.int64, ret_typ="value")
     assert l.sum() == np.sum(np.arange(0, SMALL_Y))
 
+def test_exponent_logarithm_operators():
+    a = 2*nd.ones(shape=(LARGE_X, SMALL_Y))
```

Comment:
reuse create_2d_tensor?

Reply:
a. create_2d_tensor uses 114G (htop reading) for the creating an nd array while same MXNet nd would do it around 40G.
b. np.arange is not really necessary. All we need to do is test if the function works for large arrays.
What do you think?

Reply:
maybe we should change create_2d_array to make it more efficient.

Reply:
yes. but ill address that in separate PR if that's fine.

Reply:
Yeah lets remove unnecessary numpy APIs from this test file if they eat up too much memory